.dockerignore

```diff
@@ -3,7 +3,6 @@ volume
 test_volume
 */test_volume
 volume
-.git
```

Comment:
My `.git` directory is 502MB. This is 502 MB that will be push from docker to docker every time I run any `docker build` command with this change. Why is this change necessary?

Reply:
does that happens every time you run `docker build` or just once? This allowed me to use git from inside the container. I'm gonna try to see if there's a way to add this file specifically for my image.

Reply:
Every time you run build Docker sends the whole context over a stream to the docker daemon and then looks at what is so it can tell where the cache needs to be busted at. You could probably mount the .git directory as a volume though.